packages/react/src/utils/prefers-reduced-motion/index.ts

```diff
@@ -0,0 +1,19 @@
+import { isBrowser } from '../is-browser';
+
+/**
+ * A utility to check whether a user has 'prefers-reduced-motion' enabled
+ *
+ * @return {Boolean}
+ */
+const prefersReducedMotion = (): Boolean => {
+  const reducedMotionEnabled =
+    window.matchMedia && window.matchMedia('(prefers-reduced-motion: reduce)');
+
+  if (!reducedMotionEnabled) {
+    return true;
```

Comment:
If `window` (e.g., apps using SSR) is not available or `window.matchMedia` is otherwise undefined (e.g., in tests), this will err on the side of caution and assume the user has it enabled. I recommend defaulting to `true`, but we could possibly improve this by adding a `useEffect` to check `window` once the component is mounted/window is available.